contracts/staking/contracts/src/staking_pools/MixinStakingPoolRewards.sol

```diff
@@ -308,10 +299,9 @@ contract MixinStakingPoolRewards is
     /// @param poolId Unique id of pool.
     /// @param unsyncedStake Unsynced delegated stake to pool by owner
     /// @param currentEpoch The epoch in which this call is executing
-    /// @param unfinalizedMembersReward Unfinalized total members reward
-    ///        (if any).
+    /// @param unfinalizedMembersReward Unfinalized total members reward (if any).
     /// @param unfinalizedMembersStake Unfinalized total members stake (if any).
-    /// @return totalReward Balance in ETH.
+    /// @return reward Balance in WETH.
     function _computeDelegatorReward(
```

Comment:
This wasn't introduced in this PR, but the `currentEpoch` param shadows the `currentEpoch` state variable. Also, it looks like we never actually pass in anything other than the `currentEpoch` state variable, so I think we can just remove this param.

Reply:
Don't forget to set it to a stack variable first though 😛